packages/react/src/themes/teams/components/Checkbox/checkboxVariables.ts

```diff
@@ -1,22 +1,75 @@
+  checkboxCheckedBackground: string
+  checkboxCheckedBorderColor: string
+  checkboxCheckedIndicatorColor: string
+
+  toggleBorderRadius: string
+  toggleIndicatorColor: string
+  toggleMargin: string
+  togglePadding: string
+  toggleCheckedPadding: string
+
+  disabledColor: string
+  disabledCheckboxBackground: string
+  disabledCheckboxBackgroundChecked: string
+  disabledCheckboxBorderColor: string
+  disabledCheckboxCheckedIndicatorColor: string
+  disabledToggleIndicatorColor: string
+}
+
+const [toggleMovementDistance, checkboxPadding] = [20, 2].map(v => pxToRem(v))
 
 export default (siteVars: any): CheckboxVariables => ({
-  checkboxBorderColor: siteVars.colors.grey[750],
-
-  toggleColor: siteVars.colors.brand[600],
-
-  checkedCheckboxBackground: siteVars.colors.brand[600],
-  checkedCheckboxBorderColor: siteVars.colors.brand[600],
-  checkedCheckboxColor: siteVars.colors.white,
-  checkedToggleBackground: siteVars.colors.brand[600],
-  checkedToggleBorderColor: siteVars.colors.brand[600],
-  checkedToggleColor: siteVars.colors.white,
-
-  disabledColor: siteVars.colors.grey[250],
-  disabledCheckboxBackground: siteVars.colors.grey[250],
-  disabledCheckboxBorderColor: siteVars.colors.grey[250],
-  disabledToggleBackground: siteVars.colors.grey[250],
-  disabledToggleBorderColor: siteVars.colors.grey[250],
+  checkboxTextColor: siteVars.colorScheme.default.foreground1,
+  checkboxBackground: siteVars.colorScheme.default.background,
+  checkboxBorderColor: siteVars.colorScheme.default.foreground1,
+  checkboxBorderStyle: 'solid',
+  checkboxBorderRadius: pxToRem(3),
+  checkboxBorderWidth: pxToRem(1),
+  checkboxIndicatorColor: 'transparent',
+  checkboxGap: pxToRem(12),
+  checkboxMargin: `${pxToRem(20)}, 0`,
+  checkboxPadding,
+  rootPadding: '3px 5px',
+
+  checkboxTextColorHover: siteVars.colorScheme.default.foreground,
+  checkboxBorderColorHover: siteVars.colorScheme.default.foreground,
+  checkboxCheckedBackgroundHover: siteVars.colors.brand[600],
+
+  toggleBorderRadius: pxToRem(999),
+  toggleIndicatorColor: siteVars.colors.brand[600],
+  toggleMargin: `${pxToRem(20)}, 0`,
+  togglePadding: `${checkboxPadding} ${toggleMovementDistance} ${checkboxPadding} ${checkboxPadding}`,
+
+  checkboxCheckedTextColor: siteVars.colorScheme.default.foreground,
+  checkboxCheckedBackground: siteVars.colors.brand[600],
+  checkboxCheckedBorderColor: 'transparent',
+  checkboxCheckedIndicatorColor: siteVars.colors.white,
+  toggleCheckedPadding: `${checkboxPadding} ${checkboxPadding} ${checkboxPadding} ${toggleMovementDistance}`,
+
+  disabledColor: siteVars.colorScheme.default.foregroundDisabled1,
+  disabledCheckboxBackground: siteVars.colorScheme.default.background,
+  disabledCheckboxBackgroundChecked: siteVars.colorScheme.default.backgroundDisabled,
+  disabledCheckboxBorderColor: siteVars.colorScheme.default.foregroundDisabled1,
+  disabledCheckboxCheckedIndicatorColor: siteVars.colorScheme.default.foregroundDisabled1,
```

Comment:
`disabledCheckboxCheckedIndicatorColor` should be `foregroundDisabled` instead of `foregroundDisabled1`. then you can remove from the high contrast variables file.

Reply:
I've removed most, however the checkboxBorderColorHover in this file is still necessary.